hoomd_benchmarks/common.py

```diff
@@ -119,30 +122,34 @@ def execute(self):
             print(f'.. warming up for {self.warmup_steps} steps')
         self.run(self.warmup_steps)
 
-        if (isinstance(self.device, hoomd.device.GPU)
-                and hasattr(self.sim.operations, 'is_tuning_complete')):
+        if isinstance(self.device, hoomd.device.GPU) and hasattr(
+            self.sim.operations, 'is_tuning_complete'
+        ):
             while not self.sim.operations.is_tuning_complete:
                 if print_verbose_messages:
-                    print('.. autotuning GPU kernel parameters for '
-                          f'{self.warmup_steps} steps')
+                    print(
+                        '.. autotuning GPU kernel parameters for '
+                        f'{self.warmup_steps} steps'
+                    )
                 self.run(self.warmup_steps)
 
         if print_verbose_messages:
-            print(f'.. running for {self.benchmark_steps} steps '
-                  f'{self.repeat} time(s)')
+            print(
+                f'.. running for {self.benchmark_steps} steps ' f'{self.repeat} time(s)'
+            )
 
         # benchmark
         performance = []
 
         if isinstance(self.device, hoomd.device.GPU):
             with self.device.enable_profiling():
-                for i in range(self.repeat):
+                for _i in range(self.repeat):
                     self.run(self.benchmark_steps)
                     performance.append(self.get_performance())
                     if print_verbose_messages:
                         print(f'.. {performance[-1]} {self.units}')
         else:
-            for i in range(self.repeat):
+            for _i in range(self.repeat):
```

Comment:
Did `ruff` make this change? If so, should we let a formatter change variable names?

Reply:
Yes, it was one of the unsafe fixes that I explicitly opted into on the command line `--unsafe-fixes` and then carefully reviewed (https://docs.astral.sh/ruff/rules/unused-loop-control-variable/). Running `pre-commit run` will not make these changes. 

This particular change solves https://docs.astral.sh/ruff/rules/unused-loop-control-variable/. I don't particularly agree with underscore prefixes as a solution to solve unused variables, but when switching to standard tools we are opting into what is considered the "standard" solution.